TIC-TAC-TOE/multiplayer.js

```diff
@@ -0,0 +1,225 @@
+document.onclick = function (event) {
+    if (event.target.nodeName === 'BUTTON') {
+        gamestart(event.target.id);
+    }
+}
+
+
+var tally=0;
+//Pick and Start the Model
+function gamestart(id){
+	console.log(id);
+
+	tally=tally+1;
+	//document.write(tally);
+	if(tally%2 == 0){
+	document.getElementById(id).innerHTML = "O";
+	checkO("O");
+	stopbutton("O")
+	}else{
+	document.getElementById(id).innerHTML = "X";
+	checkX("X")
+	stopbutton("X")
+	}
+	if(tally == 9){
+	document.getElementById("alertbox").innerHTML = "<h2>DRAW</h2><a href='double.html'><button class='alertbutton'>Play Again</button></a>";
+	}
+}
+
+/*-----------------------------------------------------------------------------------*/
+
+
```

Comment:
"Play Again" does not work. You still need to replace `single.html` with `singleplayer.html` everywhere. I would suggest you to revert back and change the filename to `single.html`. It will be less tedious.